### DISCUSSION TITLE
Code Submission: The Moment of Truth (and Mild Panic!) ;)

### DISCUSSION THREAD

**Discussion by WBurggraaf** [Green-Software-Foundation/hack#94] [Project Support]
Hi GSF Carbon Hack 24 team,

Excited about the hackathon! To streamline our submission, could you please clarify the following:

**Submission** (current link is broken)

**Code**: How should we package and submit code spanning 4+ GSF repos and custom code?
**Documentation**: Preferred method for supporting documents (PDFs, etc.)?
**Media**: Any preferred or restricted media formats?

**Judging**
**Scope**: Will multi-part projects be evaluated holistically or by individual components? 
Will the submission be run for testing, or only reviewed based on source code and data?
**Media**: How will media elements factor into judging alongside core functionality?

**Our Goals**
We'd appreciate a brief process outline to aid with the following:
✅ Internal Organization: Optimal code repo structure for submission.
✅ Review Strategy:  Best practices for pre-submission review.

Thanks!

**Comment by NAMRATA-WOKE** [comment]
@jmcook1186 @jawache @russelltrow to help answer some of these questions!

**Comment by russelltrow** [comment]
Hi @WBurggraaf thank you so much for reaching out and being so proactive! 

We are actually working on the submission process this week and expect to have it finalised next week. Once it's ready I'll share and make sure your questions are covered.

**Comment by srini1978** [comment]
Any updates on this ? @russelltrow

**Comment by russelltrow** [comment]
Hey @WBurggraaf @srini1978 the submission process is here: https://github.com/Green-Software-Foundation/hack/wiki/Submit-your-project-for-judging 

I will send the same over email incase you have any further questions.
